critiquebrainz/frontend/templates/navbar.html

```diff
@@ -63,3 +64,57 @@
     </div>
   </div>
 </nav>
+
+<script type="text/javascript">
+
+function driverFunction(media_query) {
+  // @media (min-width: @grid-float-breakpoint)
+  if (window.matchMedia("(min-width: 992px)").matches) {
+    searchBox.style.display = 'none';
+    searchBox.setAttribute("onblur", "focusout()");
+    typeSelect.setAttribute("onchange", "select()");
+  }
+  else {
+    searchBox.style.display = 'inline-block';
+    searchBox.removeAttribute("onblur");
+    typeSelect.removeAttribute("onchange");
+  }
+}
+
+function select() {
```

Comment:
Can the name of this function be more descriptive? Can you provide some documentation in the comment that tells what this function is for?